report_label/__manifest__.py

```diff
@@ -0,0 +1,25 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Report Labels",
+    "version": "14.0.1.0.0",
+    "summary": "Print configurable self-adhesive labels reports",
+    "author": "Iván Todorovich, Moka Tourisme, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/reporting-engine",
+    "license": "AGPL-3",
+    "category": "Reporting",
+    "maintainers": ["ivantodorovich"],
+    "depends": [
+        "base",
```

Comment:
@StefanRijnhart I have a problem launching an update --all

Adding 'base_automation' here solves the problem, but I haven't found the root problem.

![image](https://user-images.githubusercontent.com/19529533/145413232-59d6fc93-1254-4350-a4d4-30f0c94febee.png)

Reply:
Thanks, I'll have a look